packages/mui-material/src/MenuItem/MenuItem.js

```diff
@@ -243,7 +243,7 @@ MenuItem.propTypes /* remove-proptypes */ = {
    * The component used for the root node.
    * Either a string to use a HTML element or a component.
    */
-  component: PropTypes.elementType,
+  component: PropTypes /* @typescript-to-proptypes-ignore */.elementType,
```

Comment:
What are these changes for?:

```diff
-component: PropTypes.elementType,
+component: PropTypes /* @typescript-to-proptypes-ignore */.elementType,

Reply:
if not for above ignore comment, `yarn proptypes` is converting above proptype to this.  
 `component: PropTypes.oneOfType([PropTypes.oneOf(['a']), PropTypes.elementType]),`, as i', not aware of the internals of `yarn proptypes` i've added ignore comment